### PR TITLE
Feature/remove url routing

### PR DESCRIPTION
**Pull request by hjheath**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://103-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Welcome/index.js [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+          onClick={(event) => handleClick(event, nextCallback)}>
           {nextButton}
-        </Link>
+        </a>
       </div>
     </div>
   )
 }
 
+const handleClick = (event, nextCallback) => {
```
Comment (rfreitas) [inline src/components/Welcome/index.js]: if you want to get terser you could do `nextCallback => event => {`
then use it like this `onClick={handleClick(nextCallback)}`

File: src/components/Confirm/index.js [hunk 2]
```diff
@@ -30,7 +29,7 @@ const Capture = ({capture:{image, file}}) =>
   </div>
 
 
-const Previews = ({capture, nextLink, retakeAction, confirmAction} ) =>
+const Previews = ({capture, nextCallback, retakeAction, confirmAction} ) =>
```
Comment (rfreitas) [inline src/components/Confirm/index.js]: `nextCallback` and `confirmAction` are actually the same

File: package.json [hunk 3]
```diff
@@ -116,7 +116,6 @@
     "redux": "^3.5.2",
     "supports-webp": "^1.0.3",
     "visibilityjs": "^1.2.4",
-    "supports-webp": "^1.0.3",
```
Comment (hjheath) [inline package.json]: no - it seems to be duplicated on line 117 so it's fine to remove it

**Comment by rfreitas** [comment]
Travis automatic deployment: https://103-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://103-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://103-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**